### DISCUSSION TITLE
ios - communication notifications

### DISCUSSION THREAD

**Discussion by AmirDiafi** [invertase/react-native-firebase#6104] [Ideas] [answered]
is there a way to implement communication notifications on iOS ?

> Communication notifications provide a rich experience for direct communications. These notifications have a distinct user experience that features prominent avatars and group names. Communication notifications are unique in their ability to break through the scheduled notification summary by default, and can also break through a Focus.
[docs](https://developer.apple.com/documentation/usernotifications/implementing_communication_notifications
<img width="194" alt="Screen Shot 2022-03-02 at 3 11 13 PM" src="https://user-images.githubusercontent.com/59441376/156377816-00efd2d5-e75b-4332-9b9c-3f2be6fb97b9.png">
)

**Comment by mikehardy** [comment, marked as answer]
Perhaps, but definitely not here. If this happens, it will be in notifee.app

**Reply by AmirDiafi** [reply]
ah okay, thanks mike hardy
